front/src/modules/ui/editable-field/components/EditableFieldDisplayMode.tsx

```diff
@@ -16,17 +16,11 @@ export const EditableFieldNormalModeOuterContainer = styled.div<
   height: ${({ isDisplayModeFixHeight }) =>
     isDisplayModeFixHeight ? '16px' : 'auto'};
   min-height: 16px;
-
   overflow: hidden;
-
   padding: ${({ theme }) => theme.spacing(1)};
 
   ${(props) => {
-    if (props.isDisplayModeContentEmpty) {
-      return css`
-        min-width: 50px;
```

Comment:
No need to put a min-width anymore, also this will make sure the hover background correctly fits the word "Empty"